stress_test_model_eq.R

```diff
@@ -488,8 +490,12 @@ for (i in seq(1, nrow(transition_scenarios))) {
     select(
       investor_name, portfolio_name, company_name, ald_sector, technology,
       scenario_geography, year, plan_carsten, plan_sec_carsten
-    ) %>%
-    distinct(across(everything()))
+    )
+
+  report_duplicates(
```

Comment:
here quite a lot of duplicates are reported, again hopefully due to inconsistencies of faulty import.